mysite/checkers/consumers.py

```diff
@@ -0,0 +1,64 @@
+# chat/consumers.py
```

Comment:
can you add some commets to this code what is this new class for and what does it do? also new classes should have unit tet

Reply:
test*

Reply:
I still need to learn unit test and add it.

Reply:
ok no worries we can write another story for that and then go over unit test later